js/api/Geofencing.js

```diff
@@ -57,26 +55,24 @@ export function setupGeofences(callback) {
           notifyOnEntry: true,
           notifyOnExit: true,
           notifyOnDwell: true,
-          loiteringDelay: 30000
-        }
+          loiteringDelay: 30000,
+        };
+      });
 
+      console.log(`Adding ${triggerGeofences.length} geofences...`);
+      supressNotificationsTimestamp = Date.now() + 5000;
+      BackgroundGeolocation.addGeofences(triggerGeofences, () => {
```

Comment:
I haven't had any issues with this in the 2.0.0 version so far.
As long as the lib was configured, the geofences were safe to being added.